cfgov/jinja2/v1/_includes/atoms/input.html

```diff
@@ -0,0 +1,58 @@
+{# ==========================================================================
+
+   form.input()
+
+   ==========================================================================
+
+   Description: Builds form text field.
+
+   label:       Name of the field.
+
+   type:        Type of field. Will default to text, but can change to other
+                types to take advantage of HTML5 validation.
+                Possible fields: text, tel, email, url, number, date.
+
+   size:        The size of the field. Defaults to 1 (full-width).
+                Possible options: 1, 1-2, 1-4
+
+   required:    Whether the field is required. Defaults to false.
+
+   disabled:    Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+{% if not value %}
+    {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+    {% do global_dict.update({
+        'input': {
+            'label':        'This is a label',
+            'type':         'text',
+            'helperText':   lipsumText,
+            'placeholder':  'mm/dd/yyy',
+            'size':         '1-2',
+            'required':     true,
+            'disabled':     false
+        }
+    }) %}
+    {% set value = global_dict.input %}
+{% endif %}
+
+{% from 'macros/util/format/url.html' import slugify as slugify %}
+{% set id = get_unique_id('input_', '_') ~ slugify( value.label ) %}
+{% set ht_id = get_unique_id('input_ht_', '_') ~ slugify( value.label ) %}
+<div class="form-l_col form-l_col-{{ value.size if value.size else '1'}}">
+    <label class="form-label-header" for="{{ id }}">
+      {{ value.label }}
+      {%- if value.required == false -%}
+          <span class="micro-copy">&nbsp;(Optional)</span>
+      {%- endif -%}
+    </label>
+    {%- if value.helperText -%}
+        <p class="micro-copy" id="{{ ht_id }}">{{ value.helperText | safe }}</p>
+    {%- endif -%}
+    <input id="{{ id }}"
+           type="{{ value.type }}"
+           {{ 'required' if value.required else '' }}
+           {{ 'disabled' if value.disabled else ''}}
+           {%- if value.helperText -%} aria-describedby="{{ ht_id }}" {%- endif -%}
```

Comment:
These can be single line conditionals to match the required and disabled ones.

`{{ 'aria-describedby="' ~ ht_id ~ '"' if value.helperText else '' }}`